int32[] id
geometry_msgs/Vector3[] robot_positions
sensor_msgs/PointCloud2[] clouds
geometry_msgs/Vector3[] ransac_normal
geometry_msgs/Vector3[] axis
float64[] radius